tools/test.py

```diff
@@ -102,10 +102,13 @@ def main():
 
     # build the dataloader
     # TODO: support multiple images per gpu (only minor changes are needed)
+    samples_per_gpu = 1
+    if 'samples_per_gpu' in cfg.data.test:
+        samples_per_gpu = cfg.data.test.pop('samples_per_gpu')
```

Comment:
Please merge with master so that `samples_per_gpu` can be specified through command line.